modules/passkey/contracts/SafeWebAuthnSignerProxy.sol

```diff
@@ -0,0 +1,56 @@
+// SPDX-License-Identifier: LGPL-3.0-only
+pragma solidity >=0.8.0;
+import {P256} from "./libraries/WebAuthn.sol";
+
+/**
+ * @title WebAuthn Safe Signature Validator
+ * @dev A proxy contracy that points to Safe signature validator implementation for a WebAuthn P-256 credential.
+ * @custom:security-contact bounty@safe.global
+ */
+contract SafeWebAuthnSignerProxy {
+    /**
+     * @notice The X coordinate of the P-256 public key of the WebAuthn credential.
+     */
+    uint256 internal immutable X;
```

Comment:
I would try and see if using `private` makes a difference (as Solidity will not have to generate functions for these).

Reply:
Also, in terms of naming convention, we prefix `internal` and `private` members with an underscore `_`.

Reply:
> I would try and see if using private makes a difference (as Solidity will not have to generate functions for these).
The byte codesize remains same after marking private. As per documentation, getter function is generated only for `public` variables.
https://docs.soliditylang.org/en/latest/contracts.html#state-variable-visibility